app/src/main/java/com/example/vmedvediev/ua21summerdancecamp/model/NotesDatabaseHelper.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.vmedvediev.ua21summerdancecamp.model
+
+import io.realm.Realm
+
+object NotesDatabaseHelper {
```

Comment:
I think this class should be as Storage and it needs mapper from/to Realm models